cli/data.py

```diff
@@ -120,7 +120,7 @@ def update(
     else:
         multiregion_dataset = timeseries.MultiRegionDataset.from_compressed_pickle(
             dataset_utils.COMBINED_RAW_PICKLE_GZ_PATH
-        )
+        ).get_subset(state=state, fips=fips)
```

Comment:
This was a drive-by improvement to make `./run.py data update` respect the `--state` and `--fips` arguments when you pas `--no-refresh-datasets`.